netbout-web/src/main/java/com/netbout/rest/bout/TkIndex.java

```diff
@@ -79,10 +80,12 @@ public Response act(final Request req) throws IOException {
         );
         final Bout bout = new RqBout(this.base, req).bout();
         final Href home = new Href("/b").path(bout.number());
+        final RqWithDefaultHeader reqdef =
```

Comment:
@pinaf why? I'm not violating anything here - there too many variable name like 'req' etc